variables.tf

```diff
@@ -272,3 +272,9 @@ variable "tags" {
   default     = {}
   description = "A mapping of tags to assign to the bucket."
 }
+
+variable "transition_default_minimum_object_size" {
```

Comment:
could you add some validation, like this one? https://github.com/schubergphilis/terraform-aws-mcaf-aurora/blob/master/variables.tf#L401